index.js

```diff
@@ -318,7 +318,7 @@ module.exports = input => {
 		if (check([0x57, 0x41, 0x56, 0x45], {offset: 8})) {
 			return {
 				ext: 'wav',
-				mime: 'audio/x-wav'
+				mime: 'audio/wav'
```

Comment:
The official MIME-type is: `audio/vnd.wave`

See: https://en.wikipedia.org/wiki/WAV and https://tools.ietf.org/html/rfc2361

Reply:
Seems we also need to change `video/x-msvideo` to `video/vnd.avi`

Reply:
You are right!